src/main/java/mate/academy/Main.java

```diff
@@ -6,16 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger();
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Can't login ", e);
```

Comment:
The log message is not informative enough. It is recommended to include parameters or user-related information to help identify the issue more easily. For example: logger.error("Can't login user with login: {}", login, e);

Reply:
not fixed